server/src/main/java/org/opensearch/search/backpressure/stats/SearchBackpressureStats.java

```diff
@@ -47,6 +50,12 @@ public SearchBackpressureStats(StreamInput in) throws IOException {
         } else {
             searchTaskStats = null;
         }
+
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            isNodeUnderDuress = in.readOptionalBoolean();
+        } else {
+            isNodeUnderDuress = null;
+        }
```

Comment:
I'm not in favor of adding `isNodeUnderDuress` here. This is a property of the cluster, not just search backpressure. So it doesn't seem like the right place to have it.

I'm being overly cautious here as adding anything to ser/de is a one-way door decision.

Reply:
Whoops, my bad. I wanted to say node. You're right.

But the point being, search backpressure consumes this metric but doesn't produce it, so it doesn't belong here.